pinot-common/src/main/java/org/apache/pinot/common/utils/config/QueryOptionsUtils.java

```diff
@@ -438,4 +438,11 @@ private static IllegalArgumentException longParseException(String optionName, @N
     return new IllegalArgumentException(
         String.format("%s must be a number between %d and 2^63-1, got: %s", optionName, minValue, optionValue));
   }
+
+  public static String getConfigKey(Object property) {
```

Comment:
Suggest moving this next to `resolveCaseInsensitiveOptions()` and rename it to `resolveCaseInsensitiveKey()`. We should annotate this method as `Nullable`, and `resolveCaseInsensitiveOptions()` can call this method to resolve each individual option

Reply:
Handled as for the suggestion